source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -301,10 +284,10 @@ void BioXASSideTableCoordinator::onPitchControlValueChanged(double angle)
 
 void BioXASSideTableCoordinator::onPitchStopControlValueChanged()
 {
-	if(!connectedOnce_)
+	if(!initialized_)
 		return;
 
-	if (softIOCPitchStopControl_->withinTolerance(1)) {
+	if (softIOCPitchStatusControl_->isMoving()) {
```

Comment:
Is this doing what we want it to do?  Wouldn't this only be true if the status control is specifically being switched (ie: from MOVE ACTIVE to MOVE DONE)?  The status control being in a ```isMoving()``` doesn't imply that the pitch (in this case) is moving and should be stopped.  Or am I missing something here?